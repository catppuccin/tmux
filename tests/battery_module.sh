#!/usr/bin/env bash

script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
# shellcheck disable=SC1091
source "${script_dir}/helpers.sh"

# Tests that the default options are set correctly
tmux source "${script_dir}/../gruvbox_options_tmux.conf"
tmux source "${script_dir}/../gruvbox_tmux.conf"

print_option @gruvbox_status_battery
