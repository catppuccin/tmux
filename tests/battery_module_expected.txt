@catppuccin_status_battery #[fg=#b4befe]#[fg=#11111b,bg=#b4befe]#{l:#{battery_icon}} #[fg=#cdd6f4,bg=#313244]#{E:@catppuccin_battery_text}#[reverse]█#[noreverse]
